leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -236,18 +236,21 @@ var ElementInspector = Backbone.View.extend({
      * If not, set the function options so that all options are availible
     */
     checkInitialSatValue: function() {
-        // Hide all of the HTML except for the init-sat-value and function-type
-        console.log(this.intention.get('initialValue'));
-        this.$('#markedValue').hide();
-        this.$('#user-constraints').hide();
-        this.$('#segment-add').hide();
-        this.$('#constraint-repeat').hide();
-        this.$('#constraint-restart').hide();
         // Set correct dropdown options for function type based on initial satisfaction value
         $('option').show(); // Clear the previous selection
         if (this.intention.get('initialValue') == '(no value)'){
             // Hide all of the function options except for Stochastic is initial satisfaction value is '(no value)'
             $('option.B').hide(); 
+            this.$('#markedValue').hide();
+            this.$('#user-constraints').hide();
```

Comment:
I am pretty sure the user constraints div holds the segment add/constraint stuff, so I think you can just show/hide the user-constraints since that should also include the following 3 items